File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.cpp
// Last updated: 4/11/2026, 3:52:52 PM
1class Solution {
2public:
3    int removeDuplicates(vector<int>& nums) {
4        if(nums.size()==0){
5            return 0;
6        }
7
8        int i=0;
9        for(int j=0;j<nums.size();j++){
10            if(nums[j]!=nums[i]){
11                i++;
12                nums[i]=nums[j];
13            }
14        }
15        return i+1;
16    }
17};

File: 26-RemoveDuplicatesfromSortedArray/26-RemoveDuplicatesfromSortedArray.js
// Last updated: 4/2/2026, 3:07:39 PM
1/**
2 * @param {number[]} nums
3 * @return {number}
4 */
5var removeDuplicates = function(nums) {
6    if(nums.length===0){
7        return 0;
8    }
9
10    let i=0;
11    for(let j=0;j<nums.length;j++){
12        if(nums[j]!==nums[i]){
13            i++;
14            nums[i]=nums[j]
15        }
16    }
17    return i+1;
18};